events/multiCon/jamesArnoldTaylor/JATScript.js

```diff
@@ -0,0 +1,344 @@
+//
+//  JATAvatarScript.js
+//
+// Rezzes JAT's avatars as Overlays, and handle clicking on them.
+//
+//  Created by Zach Fox on 2019-03-15
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  See accompanying README.md for usage instructions.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+(function () {
+    var BASE_RELATIVE_POSITION = {"x":20.9303035736084,"y":-11.449395179748535,"z":-50.0930061340332};
```

Comment:
Could we put this in config? If there's no BASE_RELATIVE_POSITION defined, we could simply use MyAvatar.postion. I feel like this hard coded value is specific to one domain and will be changed in the future. MyAvatar.position is a nice default.

Reply:
this hard coded value _is_ specific to one domain and _will_ be changed in the future - this is a script specific to an event, and specific to one person. i don't think it's necessary _for this script_ to put something like this in the config file.